tests/testthat/test-archive.R

```diff
@@ -0,0 +1,24 @@
+library(dplyr)
+
+test_that("first input must be a data.frame",{
+  expect_error(as_epi_archive(c(1,2,3)))
```

Comment:
For the `expect_error`s here and below, we want to make sure that we're actually getting stopped by the checks that we expect to fail.  `expect_error` has at least a couple optional arguments to help here: `regexp` and `class`.  At least for the test above, only `regexp` is helpful, because we haven't assigned a custom condition class to the error message:
```r
Abort("`x` must be a data frame.")
```
However, in places where we pass a `class` argument to `Abort` in the checks, we can pass a `class` argument to `expect_error` to only match that class of errors.